CTS_XML_TEI/perseus/pdllex/lat/ls/lat.ls.perseus-eng2.xml

```diff
@@ -30401,8 +30401,8 @@ First cut at header to Lewis and Short.
+<entryFree key="mungo" type="hapax" id="n29985a"><orth lang="la" extent="full">mungo</orth>, <itype>ere</itype> <etym>obsolete, the orig. form whence emungo, mucus, mucere, mucor, mucedo; cf. Sanscr. muk, to let go, throw off; Gr. <foreign lang="greek">ἀπομύσσω</foreign>, to wipe away; <foreign lang="greek">μυκτήρ</foreign>, nose; <foreign lang="greek">μύξα</foreign>, snivel</etym>, <sense level="1" n="I" id="n29985.3"><hi rend="ital">to blow the nose</hi>: mungo, <foreign lang="greek">μύσσω</foreign>, Gloss. Philox.</sense></entryFree>
```

Comment:
There is a sense level dependency here that should perhaps be modified:
`<sense level="1" n="I" id="n29985.3">...</sense>`
It is no longer the third sense in entry n29985 and needs to be grouped with n29985a.
I will need to research the print and the current practice to determine the correct level attributions as none of this is in front of me at present.

Reply:
The original text doesn't have any sense headings here at all, but I see many digitized entries in the XML contain sense <sense level="1" n="I" id="nX.0"> for the definition part.

So in this case I would think it should be
``` 
<sense level="1" n="I" id="n29985a.0">
```
Let me know what you think.

Reply:
@nkprasad12 I agree. I think only the new entry designation (the "a") is what is missing in both cases.